zScanner/Services/Network/Medicalc/Model/DocumentTypeNetworkModel.swift

```diff
@@ -28,7 +28,8 @@ extension DocumentTypeNetworkModel {
     func toDomainModel() -> DocumentTypeDomainModel {
         return DocumentTypeDomainModel(
             id: id,
-            name: display
+            name: display,
+            subtypes: subtypes?.map({ DocumentSubTypeDomainModel(id: $0.id, name: $0.display) }) ?? []
```

Comment:
Use `subtypes?.map({ $0.toDomainModel })

The subtype should know how to map itself. Type should not be creating Instance of SubType

Reply:
Should be same as in TypeDatabaseModel

Reply:
It won't work because it can't convert DocumentTypeDomainModel to DocumentSubTypeDomainModel. Should I create a subtype network model?

Reply:
Yes :)